src/main/kotlin/io/csbroker/apiserver/controller/v2/ProblemControllerV2.kt

```diff
@@ -24,4 +30,15 @@ class ProblemControllerV2(
 
         return ApiResponse.success(findProblemDetail)
     }
+
+    @PostMapping("/{id}/challenge")
+    fun challengeProblem(
+        @LoginUser loginUser: User,
+        @PathVariable("id") id: Long,
+        @Valid
+        @RequestBody challengeCreateRequest: ChallengeCreateRequest,
```

Comment:
🚫 **[ktlint]** <annotation> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Annotation must be placed on separate line